File: delivery/src/main/java/com/lgior/TestApplication.java
package com.lgior;

import com.lgior.resources.HelloFromKotlin;
import com.lgior.resources.HelloWorldResource;

import io.dropwizard.Application;
import io.dropwizard.setup.Bootstrap;
import io.dropwizard.setup.Environment;

public class TestApplication extends Application<TestAppConfiguration> {

    public static void main(final String[] args) throws Exception {
        new TestApplication().run(args);
    }

    @Override
    public String getName() {
        return "Prueba dropwizard";
    }

    @Override
    public void initialize(final Bootstrap<TestAppConfiguration> bootstrap) {
        // TODO: application initialization
    }

    @Override
    public void run(final TestAppConfiguration configuration,
                    final Environment environment) {
		HelloWorldResource resource = new HelloWorldResource();
        HelloFromKotlin helloFromKotlin = new HelloFromKotlin();

        environment.jersey().register(resource);
        environment.jersey().register(helloFromKotlin);
	}

}

File: delivery/src/main/kotlin/com/lgior/app/TestApplication.kt
package com.lgior.app

import com.lgior.resources.HelloFromKotlin
import com.lgior.resources.HelloWorldResource
import io.dropwizard.Application
import io.dropwizard.setup.Bootstrap
import io.dropwizard.setup.Environment

class TestApplication : Application<TestAppConfiguration?>() {
    override fun getName(): String {
        return "Prueba dropwizard"
    }

    override fun initialize(bootstrap: Bootstrap<TestAppConfiguration?>) {
        // TODO: application initialization
    }

    override fun run(configuration: TestAppConfiguration?,
                     environment: Environment) {
        val resource = HelloWorldResource()
        val helloFromKotlin = HelloFromKotlin()
        environment.jersey().register(resource)
        environment.jersey().register(helloFromKotlin)
    }

    companion object {
        @Throws(Exception::class)
        @JvmStatic
        fun main(args: Array<String>) {
            TestApplication().run(*args)
        }
    }
}